components/concept/storage/src/main/java/mozilla/components/concept/storage/HistoryStorage.kt

```diff
@@ -201,13 +201,15 @@ enum class FrecencyThresholdOption {
  * @property visitTime The time the page was visited in integer milliseconds since the unix epoch.
  * @property visitType What the transition type of the visit is, expressed as [VisitType].
  * @property previewImageUrl The preview image of the page (e.g. the hero image), if available.
+ * @property isRemote Distinguishes visits made on the device and those that come from syncing.
```

Comment:
nit: "... that come from Sync."

where the name Sync refers to Firefox Sync.

Reply:
fixed